tests/src/test/scala/org/http4s/EntityDecoderSpec.scala

```diff
@@ -439,4 +449,5 @@ class EntityDecoderSpec extends Http4sSpec with PendingUntilFixed {
     EntityEncoder.simple[IO, ErrorJson](`Content-Type`(MediaType.`application/json`))(json =>
       Chunk.bytes(json.value.getBytes()))
 
+  checkAll("SemigroupK[EntityDecoder[IO, ?]]", SemigroupKTests[EntityDecoder[IO, ?]].semigroupK[String])
```

Comment:
The line that motivated this entire PR.  Sad: it takes 81 seconds on my machine.